src/main/java/browserview/BrowserComponent.java

```diff
@@ -486,4 +486,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
+    public void loadTab(String currentUrl) {
+        if (isCurrentUrlIssue() && (currentUrl.endsWith("/commits") || currentUrl.endsWith("/files"))) {
+            driver.findElement(By.xpath("//a[@data-container-id='discussion_bucket']")).click();
```

Comment:
would it be better to just explicitly load the pr page instead of clicking on the tab?